docs/misc/faq-chain-transition.md

```diff
@@ -0,0 +1,81 @@
+- [Will the Network ID change?](#will-the-network-id-change-)
+- [Is Kaia a new chain with a different chain ID, or an upgrade based on the Klaytn chain?](#is-kaia-a-new-chain-with-a-different-chain-id-or-an-upgrade-based-on-the-klaytn-chain-)
+- [Is KAIA a rebrand from KLAY or a completely new token?](#is-kaia-a-rebrand-from-klay-or-a-completely-new-token-)
+- [Will Kaia be on the Klaytn mainnet or a different mainnet?](#will-kaia-be-on-the-klaytn-mainnet-or-a-different-mainnet-)
+- [Will the current Klaytn mainnet continue to work after the rebrand?](#will-the-current-klaytn-mainnet-continue-to-work-after-the-rebrand-)
+- [When will the KAIA market open?](#when-will-the-kaia-market-open-)
+- [Will Klaytn provide a swap or migration website for new KAIA tokens?](#will-klaytn-provide-a-swap-or-migration-website-for-new-kaia-tokens-)
+- [Will there be any delay in receiving KAIA tokens?](#will-there-be-any-delay-in-receiving-kaia-tokens-)
+- [Is there a limit on the amount for swapping KLAY and FNSA tokens?](#is-there-a-limit-on-the-amount-for-swapping-klay-and-fnsa-tokens-)
+- [Is it necessary to update the brand name from Klaytn to Kaia in the wallet?](#is-it-necessary-to-update-the-brand-name-from-klaytn-to-kaia-in-the-wallet-)
+- [What happens if we don't upgrade to the Kaia brand after the mainnet upgrade?](#what-happens-if-we-dont-upgrade-to-the-kaia-brand-after-the-mainnet-upgrade-)
+- [What will happen to Klaytnscope, the default explorer?](#what-will-happen-to-klaytnscope-the-default-explorer-)
+- [What support will Kaia Foundation provide for the migration of Klaytn and Finschia apps?](#what-support-will-kaia-foundation-provide-for-the-migration-of-klaytn-and-finschia-apps-)
+- [Will Kaia Foundation continue to provide code audit grants as Klaytn Foundation did?](#will-kaia-foundation-continue-to-provide-code-audit-grants-as-klaytn-foundation-did-)
+
+## As an RPC node provider, what changes do I need to make to my infrastructure to support the transition from Klaytn to Kaia, and will there be any breaking changes in the RPC APIs? <a id="as-an-rpc-node-provider-what-changes-do-i-need-to-make-to-my-infrastructure-to-support-the-transition-from-klaytn-to-kaia-and-will-there-be-any-breaking-changes-in-the-rpc-apis-"></a>
+
+Most RPC APIs will remain the same on the Kaia chain, with the exception of namespace changes from klay_ to kaia_. However, klay_ will still be available for backward compatibility.
+
+To support the transition from Klaytn to Kaia, RPC node providers will need to upgrade their binary to the Kaia version once it is released. No immediate action is required before the upgrade.
+
+## Will the Network ID change? <a id="will-the-network-id-change-"></a>
+
+No, the Klaytn Network ID will remain the same. The original URL will be active for three months to ensure a smooth transition.
```

Comment:
Please say like "Network ID (Chain ID)" because Chain ID is also widely used.

Also the URL is not directly related to Network ID I guess..?

Reply:
It looks nicer to merge the two questions!

Still, it's unclear what "The original URL" means in this context. A _chain_ doesn't have its URL. Does it mean the klaytn websites? public endpoints? or any URLs and domains maintained by the Klaytn Foundation? I think this last line needs examples of those original URLs.

e.g.
... is now referred to as "Mainnet" or "Kaia Mainnet." The original URLs related to Klaytn such as docs, website, public endpoints, and package download links will remain for three months to ensure a smooth transition.

Reply:
@blukat29 I've made an additional commit to apply feedbacks. Please check out the changes.

Reply:
Looks good. Thanks